packages/credential-showcase-openapi/openapi/openapi.yaml

```diff
@@ -2077,6 +2082,55 @@ components:
           items:
             type: string
           description: List of restrictions on the predicate
+    AcceptCredentialAction:
+      allOf:
+        - $ref: '#/components/schemas/StepAction'
+        - type: object
+          required:
+            - credentialDefinitionId
+          properties:
+            credentialDefinitionId:
+              type: string
+              description: Credential definition ID
+              example: 123e4567-e89b-12d3-a456-426614174456
+            connectionId:
+              type: string
+              description: Connection ID
+              example: 341c384b-3313-41ba-96a7-4f07329cd6b5
+    ShareCredentialAction:
+      allOf:
+        - $ref: '#/components/schemas/StepAction'
+        - type: object
+          required:
+            - credentialDefinitionId
+          properties:
+            credentialDefinitionId:
+              type: string
+              description: Credential definition ID
+              example: 123e4567-e89b-12d3-a456-426614174456
+            connectionId:
+              type: string
+              description: Connection ID
+              example: 341c384b-3313-41ba-96a7-4f07329cd6b5
+    SetupConnectionAction:
+      allOf:
+        - $ref: '#/components/schemas/StepAction'
+        - type: object
+      # TODO properties?
+    ChooseWalletAction:
+      allOf:
+        - $ref: '#/components/schemas/StepAction'
+        - type: object
+      # TODO properties?
+    ButtonAction:
+      allOf:
+        - $ref: '#/components/schemas/StepAction'
+        - type: object
+          properties:
+            goToStep:
```

Comment:
why is this needed?

Reply:
@ronal2do 
That's explained in the [ticket](https://4sure.atlassian.net/browse/SHOWCASE-220)

> We didn’t discuss BUTTON, we do not have to use it yet, but it would allow to create more complex flows where you could jump to a part with other screens and potentially other credentials.